ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -86,12 +86,24 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   const visual = merge({}, DEFAULT_VISUAL, props.spec.visual);
 
   // convert Perses dashboard format to be ECharts compatible
-  const yAxis = {
+  const yAxis: YAXisComponentOption = {
     show: y_axis?.show ?? DEFAULT_Y_AXIS.show,
-    min: y_axis?.min, // leaves min and max undefined by default to let ECharts calcualate
+    min: y_axis?.min,
     max: y_axis?.max,
   };
 
+  if (yAxis.min === undefined) {
+    // sets minimum axis label relative to data instead of zero
+    yAxis.min = (value) => {
+      // https://echarts.apache.org/en/option.html#yAxis.min
+      if (value.min <= 1) {
+        return 0;
```

Comment:
Open to removing this, but before I added this condition, the origin looked odd for PercentDecimal cases or when the min value was 1, ex of before I added it:
### Before
<img width="1122" alt="image" src="https://user-images.githubusercontent.com/9369625/230621963-04612d83-3329-4bb1-82cc-63ed7a4fbeba.png">

### After
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/9369625/230622056-05840d28-dd82-4c42-875a-27124e41b3e5.png">

Reply:
Seems reasonable to me!

Reply:
Yeah good call. These two charts definitely look better with the adjustment. We can plan to keep an eye out when we see this on charts in the wild (to continue to confirm it looks good). 

In general, I feel like it's best when we aim for the best for the overall chart viewing experience without the user having to do anything, so I like the call.